extensions/kubernetes/deployment/src/main/java/io/quarkus/kubernetes/deployment/KubernetesProcessor.java

```diff
@@ -51,7 +51,22 @@
 class KubernetesProcessor {
 
     private static final String PROPERTY_PREFIX = "dekorate.";
-    private static final String ALLOWED_GENERATOR = "kubernetes";
+    private static final Set<String> ALLOWED_GENERATORS = new HashSet<String>() {
```

Comment:
Just nitpicking here, but the codebase generally shies away from this syntax and uses the more verbose `new HashSet(Arrays.asList())`